identify_duplicate_categories.gd

```diff
@@ -0,0 +1,40 @@
+################################################################################
+# starts_with
+#
+# A helper function to see if a string starts with a paticular value or not.
+################################################################################
+static func starts_with(value: String, prefix: String):
+	if value.length() < prefix.length():
+		return false
+	return value.substr(0, prefix.length()) == prefix
+
+
+################################################################################
+# get_duplicate_categories
+#
+# Parses the stdout of the xml_converter and extracts any category colisions if
+# there are any that exist.
+################################################################################
```

Comment:
It might be worth adding an example here of what we expect to parse so that if it ever breaks, we have something to compare against. Especially since we do not have integration tests for this code.

Reply:
done